### PR TITLE
clicking on weak supervision results adds them to manual label set

### PR DESCRIPTION
**Pull request by jhoetter**
Clicking on a weakly supervised label now adds them to the manual labeling set. Addresses code-kern-ai/refinery#51.

Next step could possibly be to tab-navigate through weak supervision results, and accepting them via enter.

### CODE REVIEW

File: src/app/labeling/components/labeling.component.ts [hunk 1]
```diff
@@ -1389,7 +1393,31 @@ export class LabelingComponent implements OnInit, OnDestroy {
     }
   }
 
+  addLabelToMarkedTextFromSuggestion(overlayData) {
```
Comment (JWittmeyer) [inline src/app/labeling/components/labeling.component.ts:1396]: no checks are made for the given weak supervision label. This means that the user could accept the same label multiple times or create an overlapping issue with other manual labels (or itself). 
image

**Comment by jhoetter** [comment]
check [gateway](https://github.com/code-kern-ai/refinery-gateway/pull/46) and [submodule](https://github.com/code-kern-ai/refinery-submodule-model/pull/11)

File: src/app/labeling/components/labeling.component.ts [hunk 2]
```diff
@@ -1389,7 +1393,31 @@ export class LabelingComponent implements OnInit, OnDestroy {
...
+    const dataEntry = {
+      startIdx: overlayData.tokenStartIdx,
+      endIdx: overlayData.tokenEndIdx,
+      value: overlayData.value,
+    };
+
+    this.somethingLoading = true;
```
Comment (FelixKirsch) [inline src/app/labeling/components/labeling.component.ts:1403]: when clicking on a weak supervision label, which has an overlapping manual label is not set as expected. But the cursor has a loading spinner attached to it, which isn't removed.

**Review by FelixKirsch — APPROVED** [review]

File: src/app/labeling/components/labeling.component.ts (hunk 2 above)
Comment (jhoetter) [reply src/app/labeling/components/labeling.component.ts:1403]: saw this too, i will create an issue for this. but only saw this in minor cases

**Pull request merged**